Godeps/_workspace/src/k8s.io/kubernetes/pkg/apis/extensions/install/install.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/golang/glog"
 
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
Basically, the design of "install" being order independent due to packaging order means that unless you take this dependency you can't order installs.  So you have to have some class that registers them all in order, or add this dependency.  This is why we don't do static initializers.